internal/notif/teams/teams.go

```diff
@@ -0,0 +1,90 @@
+package teams
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/crazy-max/diun/internal/model"
+	"github.com/crazy-max/diun/internal/notif/notifier"
+)
+
+// Client represents an active webhook notification object
+type Client struct {
+	*notifier.Notifier
+	cfg       model.NotifTeams
+	app       model.App
+	userAgent string
+}
+
+// New creates a new webhook notification instance
+func New(config model.NotifTeams, app model.App, userAgent string) notifier.Notifier {
+	return notifier.Notifier{
+		Handler: &Client{
+			cfg:       config,
+			app:       app,
+			userAgent: userAgent,
+		},
+	}
+}
+
+// Name returns notifier's name
+func (c *Client) Name() string {
+	return "teams"
+}
+
+type Sections struct {
```

Comment:
Could you please add comment here ? @Andrioshe

Reply:
done 👍